### PR TITLE
Bump Substrate

### PR DESCRIPTION
**Pull request by adoerr**
Also removed old Sec advisory.

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -5911,18 +5911,18 @@ checksum = "388a1df253eca08550bef6c72392cfe7c30914bf41df5269b68cbd6ff8f570a3"
 
 [[package]]
 name = "serde"
-version = "1.0.122"
+version = "1.0.119"
```
Comment (HCastano) [inline Cargo.lock]: Why did `serde` need to be downgraded?

File: Cargo.lock [hunk 2]
```diff
@@ -6900,9 +6900,9 @@ checksum = "1e81da0851ada1f3e9d4312c704aa4f8806f0f9d69faaf8df2f3464b4a9437c2"
 
 [[package]]
 name = "syn"
-version = "1.0.60"
+version = "1.0.58"
```
Comment (HCastano) [inline Cargo.lock]: Same question here

File: deny.toml [hunk 3]
```diff
@@ -48,8 +48,6 @@ notice = "warn"
 # A list of advisory IDs to ignore. Note that ignored advisories will still
 # output a note when they are encountered.
 ignore = [
-	# yaml-rust < clap. Not feasible to upgrade and also not possible to trigger in practice.
-	"RUSTSEC-2018-0006",
```
Comment (HCastano) [inline deny.toml:52]: If it's fine to get rid of this, we should probably also do it in the `parity-bridges-common` repo

**Review by HCastano — APPROVED** [review]

File: Cargo.lock (hunk 1 above)
Comment (adoerr) [reply Cargo.lock]: I have no idea

File: Cargo.lock (hunk 2 above)
Comment (adoerr) [reply Cargo.lock]: Still no idea

**Pull request merged**